src/component/projects.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Styled from 'styled-components';
+import Gallery from 'react-photo-gallery';
+
+
+const Container = Styled.div`
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  padding-top: 3vw;
+  padding-bottom: 3vw;
+`;
+
+const FullThemeline = Styled.div`
+  border-bottom-style:  solid;
+  border-width: 1px;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  margin-top: 2vw;
+  margin-bottom: 2vw;
+`;
+
+const TagsRow = Styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const Tag = Styled.h3`
+  font-size: 1vw;
+  cursor: pointer;
+  margin-left: 10px;
+  margin-right: 10px;
+`;
+
+
+class Projects extends React.Component {
+  render() {
+    const PHOTO_SET = [
```

Comment:
In the stone age 🗿  people used all capital letters to show that it's a constant. Just to show... functionally it has no difference. But now we don't need to do this because we have `const`!!!